samples/subsys/mgmt/mcumgr/smp_svr/prj_udp.conf

```diff
@@ -0,0 +1,40 @@
+# Enable mcumgr.
```

Comment:
I think you should make the UDP support in overlay conf file (overlay-udp.conf). Then it is much easier to enable it during compilation.

Reply:
I agree, but I think this would require a rewrite of the entire sample, as there currently is no base config on which to overlay additional settings. I can do a rewrite, but I can't really verify that the bluetooth sample still works as I don't have the nrf board.

Reply:
> I agree, but I think this would require a rewrite of the entire sample, as there currently is no base config on which to overlay additional settings.

Yes, you would probably need to rethink the content of the conf files. You should place common config into prj.conf file and then just things related to features etc into overlay config file. I think this should not be a big effort and would make the sample more usable.